ffi/rtc_h.lua

```diff
@@ -0,0 +1,38 @@
+local ffi = require("ffi")
+
+ffi.cdef[[
+struct rtc_time {
+  int tm_sec;
+  int tm_min;
+  int tm_hour;
+  int tm_mday;
+  int tm_mon;
+  int tm_year;
+  int tm_wday;
+  int tm_yday;
+  int tm_isdst;
+};
+struct rtc_wkalrm {
+  unsigned char enabled;
+  unsigned char pending;
+  struct rtc_time time;
+};
+static const int RTC_WKALM_SET = 1076391951;
+static const int RTC_WKALM_RD = 2150133776;
+typedef long int time_t;
```

Comment:
This might have been tricky (Y2K38 stuff), but apparently is accurate on Kobo:

```
echo | arm-kobo-linux-gnueabihf-gcc -E -xc -include 'time.h' - | grep time_t

__extension__ typedef long int __time_t;
typedef __time_t time_t;
    __time_t tv_sec;
extern time_t time (time_t *__timer) __attribute__ ((__nothrow__ , __leaf__));
extern double difftime (time_t __time1, time_t __time0)
extern time_t mktime (struct tm *__tp) __attribute__ ((__nothrow__ , __leaf__));
extern struct tm *gmtime (__const time_t *__timer) __attribute__ ((__nothrow__ , __leaf__));
extern struct tm *localtime (__const time_t *__timer) __attribute__ ((__nothrow__ , __leaf__));
extern struct tm *gmtime_r (__const time_t *__restrict __timer,
extern struct tm *localtime_r (__const time_t *__restrict __timer,
extern char *ctime (__const time_t *__timer) __attribute__ ((__nothrow__ , __leaf__));
extern char *ctime_r (__const time_t *__restrict __timer,
extern int stime (__const time_t *__when) __attribute__ ((__nothrow__ , __leaf__));
extern time_t timegm (struct tm *__tp) __attribute__ ((__nothrow__ , __leaf__));
extern time_t timelocal (struct tm *__tp) __attribute__ ((__nothrow__ , __leaf__));
```

Plus, it didn't blow up at runtime on my H2O :D.

Reply:
I'll still pcall it of course. :-p

Reply:
Ultimately it's a pointer, so it wouldn't matter too much, I think?

Reply:
Fair enough.